test/integration/minishift.go

```diff
@@ -190,6 +190,43 @@ func (m *Minishift) executingMinishiftCommand(command string) error {
 	return nil
 }
 
+func (m *Minishift) setImageCaching(operation string) error {
+	enabled := "true"
+	if operation == "disabled" {
+		enabled = "false"
+	}
+
+	return m.executingMinishiftCommand(fmt.Sprintf("config set image-caching %s", enabled))
+}
+
+func (m *Minishift) imageExportShouldComplete(noOfImages int) error {
+	// poll till the output of the `minishift image list` shows three OpenShift cached images
+	timeout := time.NewTimer(20 * time.Minute)
+
+outerPollActive:
+	for {
+		select {
+		case <-timeout.C:
+			return errors.New("Timed out in getting the number of default cached images")
+		default:
+			cmdOut, _, _ := m.runner.RunCommand("image list")
```

Comment:
Maybe it would be safer to also check the cmdErr and exitCode. If a three line error occurs it would pass.

Reply:
@budhrg You are right. This is fine then.